### PR TITLE
Upgraded dependencies

### PR DESCRIPTION
**Pull request by renuzit**
Previously there were 12 security issues with npm modules that were
dependencies of the wallet app. After this commit, there will be only
five security issues. This commit upgrades most of the npm packages
to new versions. An effort will still need to be made to upgrade
submodules dependencies to make sure the app is free of any security
issues.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,5 @@
- {
-  "//":"Changes to this file will be overwritten",
-  "//":"        Modify it in the app-template directory",
-
+{
+  "//": "        Modify it in the app-template directory",
```
Comment (egyptianbman) [inline package.json]: I've always wondered how you add comments to `package.json`!

File: package.json [hunk 2]
```diff
@@ -49,72 +47,70 @@
...
-    "final:desktop":          "npm run build:desktop && npm run build:osx",
-    "run:android":            "cordova run android --device",
-    "run:android-release":            "cordova run android --device --release",
-    "log:android":            "adb logcat | grep chromium",
-    "sign:android":           "rm -f platforms/android/build/outputs/apk/android-release-signed-aligned.apk; jarsigner -verbose -sigalg SHA1withRSA -digestalg SHA1 -keystore ../copay.keystore -signedjar platforms/android/build/outputs/apk/android-release-signed.apk  platforms/android/build/outputs/apk/android-release-unsigned.apk copay_play && $ANDROID_HOME/build-tools/26.0.1/zipalign -v 4 platforms/android/build/outputs/apk/android-release-signed.apk platforms/android/build/outputs/apk/android-release-signed-aligned.apk",
-    "test":                   "echo \"no package tests configured\"",
-    "clean":                  "trash platforms && trash plugins && cordova prepare",
-    "unstage-package":        "git reset package.json",
-    "clean-all":              "git clean -dfx"
+    "postinstall": "sed -i  -e \"s/.*require(...\\/...);//g\" node_modules/asn1.js-rfc5280/index.js; bower install",
```
Comment (egyptianbman) [inline package.json]: I believe there are better npm implementations for scripts that we can utilize to clean this up. Low priority.

**Comment by renuzit** [comment]
This branch is having problems building android. Btczgod is working on resolving the android issues, so we will wait until he has a build working and rebase off that branch.

**Comment by egyptianbman** [comment]
Sounds good.

**Comment by renuzit** [comment]
Android issues should be resolved now.

**Pull request merged**